framework/interface/templates/plugin_report.html

```diff
@@ -33,7 +34,7 @@ <h4 id="header_badge_{{ test_code }}"></h4>
                     <div class="tab-content">
                         {% for poutput in poutput_list %}
                         {% set pkey = poutput['plugin_type'] + '_' + poutput['plugin_code'] %}
-                        <div class="tab-pane" id="{{ poutput['plugin_group'] }}__{{ poutput['plugin_type'] }}__{{ test_code }}">
+                        <div class="tab-pane{{ pactive }}" id="{{ poutput['plugin_group'] }}__{{ poutput['plugin_type'] }}__{{ test_code }}">
```

Comment:
As effect to the above change, `class="tab-pane{{ pactive }}"` becomes `class="tab-pane {{ pactive }}"`